gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -560,7 +564,7 @@ export function GeneralSettings() {
                 step={0.1}
               />
             </div>
-            <div className="flex gap-5 pt-5 md:flex-row flex-col">
+            <div className="flex gap-5 pt-5 pb-7 md:flex-row flex-col">
```

Comment:
modify the actual div that you are making please
```suggestion
            <div className="flex gap-5 pt-5 md:flex-row flex-col">
```